Description: Fix issue where REG_A0 is already defined by riscv64 platform
Author: Tien Hock Loh <tienhock.loh@starfivetech.com>
Forwarded: https://listengine.tuxfamily.org/lists.tuxfamily.org/hatari-devel/2022/06/msg00140.html

Index: hatari-2.3.1+dfsg/src/includes/m68000.h
===================================================================
--- pkg-hatari.orig/src/includes/m68000.h
+++ pkg-hatari/src/includes/m68000.h
@@ -15,6 +15,9 @@
 /*			a possible pairing between add.l (a5,d1.w),d0	*/
 /*			and move.b 7(a5,d1.w),d5.			*/
 
+#ifdef REG_A0
+#undef REG_A0
+#endif
 
 #ifndef HATARI_M68000_H
 #define HATARI_M68000_H
